rslib/src/notetype/schema11.rs

```diff
@@ -211,6 +214,9 @@ pub struct NoteFieldSchema11 {
     #[serde(default, deserialize_with = "default_on_invalid")]
     pub(crate) description: String,
 
+    #[serde(deserialize_with = "deserialize_bool_from_anything")]
```

Comment:
```suggestion
    #[serde(default)]
```

Since this is a new key, we don't need to worry about legacy code accidentally using 0/1, and we don't want to error if the key is missing.